server/functions/casestats.json

```diff
@@ -0,0 +1,2781 @@
+{
+    "jurisdictionStats": [
+        {
```

Comment:
Prefer 2 spaces for indentation but not a big issue

Reply:
Done